api/v1alpha1/workstatus_types.go

```diff
@@ -48,7 +48,6 @@ type WorkStatusList struct {
 
 // Manifest represents a resource to be deployed
 type RawStatus struct {
-	// +kubebuilder:validation:EmbeddedResource
```

Comment:
This seems like an inferior choice for how to solve the problem.

Reply:
Would be happy to learn about a better solution.